builder/fetch.py

```diff
@@ -31,8 +32,14 @@
 except:
     msvcrt = None
 
+from util import run_command
 
-MANIFEST_URL = 'https://d19elf31gohf1l.cloudfront.net/_binaries/MANIFEST'
+FETCH_URL = 'https://d19elf31gohf1l.cloudfront.net/_binaries'
```

Comment:
How stable is this?

Reply:
It won't change unless we de/re-provision cloudfront AFAIK. And maybe not even then.